src/test/java/JavAI/StorageTest.java

```diff
@@ -0,0 +1,55 @@
+package javai;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+
+public class StorageTest {
+
+    private Storage storage;
+    private final String FILE_PATH = "./src/main/txtFolder/JavAI.txt";
+
+    @BeforeEach
+    void setUp() {
+
+        storage = new Storage(FILE_PATH);
+    }
+
+    @Test
+    void testLoad() throws IOException {
```

Comment:
you may want to name your functions more explicitly - whether you expect a success / exception to be thrown.